src/services/single-pokemon.js

```diff
@@ -0,0 +1,16 @@
+export const getSinglePokemon = async (pokemon) => {
+  const response = await fetch(
+    "https://pokeapi.co/api/v2/pokemon/" + pokemon.toLowerCase()
+  );
+  if (response.status === 404) {
+    throw Error(
+      "Shoot, that Pokemon's not in the Pokedex, please enter a valid Pokemon."
+    );
+  }
+  const data = await response.json();
+
+  const singlePokemon = [
+    { name: data.name, id: data.id, image: data.sprites.front_default },
+  ];
+  return singlePokemon;
```

Comment:
```suggestion
  return { name: data.name, id: data.id, image: data.sprites.front_default };
```

<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏A `singlePokemon` suggests an object to me, not an array of one. I would simplify this function to only return a single data point. That way your component doesn't need to pull off the first item of the array.